modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_docker_container_upload.rb

```diff
@@ -27,7 +27,7 @@ def perform(claim_id) # rubocop:disable Metrics/MethodLength
         log_job_progress(claim_id,
                          "Successfully submitted to Docker container with response: #{evss_res}")
         # update with the evss_id returned
-        auto_claim.update(evss_id: evss_res[:claimId])
+        auto_claim.update!(evss_id: evss_res[:claimId]) if evss_res&.include?(:claimId)
```

Comment:
we should be raising here now with the default `raise_backend_exception` 'EVSS500' - I think this check was preventative for the potential for HTML to come back?